modules/apps/frontend-js/frontend-js-spa-web/test/app/App.js

```diff
@@ -1132,9 +1132,9 @@ describe('App', function () {
 			.cancel();
 	});
 
-	it('navigates when clicking on routed links', () => {
+	it('navigates when clicking on routed links', async () => {
 		this.app = new App();
-		this.app.addRoutes(new Route('/path', Screen));
+		this.app.addRoutes(new Route('/pathote', Screen));
```

Comment:
I noticed this typo with `/pathote` and I didn't look too far into this (super unfamiliar with all this SPA stuff), but it was concerning how this was still passing. And it seems like simply having `expect(this.app.navigationStrategy).toBe('immediate');` without any of the route setup/clicking will pass anyway so I'm not sure this is testing anything?

I could just be not understanding, but wanted to at least point it out.

Reply:
Thanks a lot, @thektan ! It's just a typo I was missing.